mitxpro/settings.py

```diff
@@ -1055,6 +1057,11 @@
     "courses.credentials.build_digital_credential"
 )
 
+# mitol-django-authenticaton
+import_settings_modules(globals(), "mitol.authentication.settings.djoser_settings")
```

Comment:
Is this separate from the other `import_settings_module` call intentionally? If yes, could you add a comment explaining why?

Reply:
The import_settings_module is used to pick up DJOSER settings defined in ol-django-authentication app. However I do not see any other reference defined for import_settings_module other than the one defined above.

Reply:
Hm, I thought we did, don't worry about it for now then